src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -31,14 +31,26 @@
         "SSID", "StudentIdentifier"
     );
 
+    private static final Map<String, String> relationshipNames = ImmutableMap.of(
```

Comment:
Only thing I don't like is that guava ImmutableMap.of() does not support more than 12 fields (6 pairs) :(